Wshaw bio

dasper opened the pull request:
# Valebant nasci arguitur nebulas maior
## Crudelis resistite exitium tantique haec

Lorem markdownum fero comas vaporem quantumque factorum, centum, gravidi terra
per venenum nondum sepulcris [sociare](http://twitter.com/search?q=haskell),
ebur. Insula longa, carmine ignem transfert Graium parvum aliqua verba rediit
Nili esse vertor citra suos.

Non dilapsa pectus, adultera carmine omen, ita captam et in dona. Acta ipsos
quaerentibus corpus, odoribus Lernaeae ensis gratissime mollire ferarum iuventa
_unda fessa_. **Et quamvis dare** saepe peperisse mutataque foedari, iuvenes
unda saevae protinus. Ne virgo Cephenum Somne, remane sororem, aut meis incisa
sit soceri mundo?

Sive inde membris succincta illud: nec petit imoque patruo nitor exhortor
_credita_ cunctantem, transierant illa Venus fratrum fores! Pectore adspicio in
tecta in lenius femina; a ventis patrio. Tamen fraternis dixit, facies narremur
potiunda inculpata in exsultat, hoc bella peterem, cornibus comitante relinquit
numen, falsamque. Pressum _silvas_ teli indiciumque ritus exponimur concursibus
maiora in frater spreta portus moderamen.
## Tunc adoratis fronte conantesque inanes

Adspergit Eurotas lacrimis nocte; imitata urbis tela dedit aethere tamquam
flexile: iam illius ira clavigeram montibus. Suspiria ventis, cum vixque
consorte se urna nec ferat suum altior ad **fervida**. Veloci meruisse, ferre
vocavit, montes inseritur hunc; educat, curas liquidarum Lucina Io virginitas
me. Fulvis Salmacis superi secutae ad intravit forma: dum cui **postquam
furtoque**, nihil, manu praestantior. Orbem nubibus aeterna, in ora moriturae
cum, in reliqui?

> Nullo in crudelis _talia curam_ inpositum _me duorum vertice_, Calydonides
> mihi gravitate ripae, sim. Cruore dextra. Vel in brevis atque perterrita malum
> facta hoc, per. Senex urbem quin audit en revelli periura omnes patria, aquis
> Iovis, et.
## Tyrrhena summis via nomina

Cum cui fugias, est parentes tenus instrumenta mittit? Quam carmina pedem. Ipse
[partim](http://html9responsiveboilerstrapjs.com/). Cur honos incessere pando
Naryciamque suum deorum cerae, sagacior plenissima et, catenis adeo et moenia
Ixione, metuam. Cortice perque premunt.

Retinere quotiens _verba sacrificos_ figura corpore placare nulla [flectitur
quoniam](http://example.com/). Inde labefactum carbasa inscribenda erat illi
periturae enim falleret [Achille coepti](http://seenly.com/) fortunam iuventa
laedere frondis, findi, tamen.
1. Omnia gelido mutatur
2. Vellet fama per exit
3. Relictas tempus reliquerat et retemptatis celer

Dixit habitantem placidos inprovisoque nautae
[formosissimus](http://www.uselessaccount.com/) agros _mortalis opusque_ murum,
in levi, luce. Dedere in gemino vulnere in sine coegerat saltus.

dasper committed:
Adding my bio to the repository.

dasper committed:
Modified wshaw's Bio

dasper committed:
Modified wshaw's Bio again

bobby-drk commented:
This is a really great bio!

geothird commented on wshaw-bio.md:
```diff
@@ -0,0 +1,8 @@
+- A
+- List
+
+**The network is slow**
```
:hankey:

jeffajohns commented:
THIS IS GREAT WHAT YOU HAVE DONE HERE!!

I LIKE YOUR COMPUTER PROGRAMMING!!

![](http://www.tattoobite.com/wp-content/uploads/2014/02/geek-computer-logos-tattoo-pictures.jpg)

dasper committed:
Made an update!


┓┏┓┏┓┃
┛┗┛┗┛┃＼○／
┓┏┓┏┓┃   /    No unit
┛┗┛┗┛┃ノ)
┓┏┓┏┓┃         tests
┛┗┛┗┛┃ 
┓┏┓┏┓┃ 
┛┗┛┗┛┃ 
┓┏┓┏┓┃         
┃┃┃┃┃┃
┻┻┻┻┻┻
